database/release_dao.py

```diff
@@ -0,0 +1,142 @@
+import sqlalchemy as sa
+from config import DATABASE_URL
+from models import Release
+from sqlalchemy.orm import sessionmaker
+
+# Needs to implement all CRUD operations on the given database.
+
+engine = sa.create_engine(DATABASE_URL)
+
+Session = sessionmaker(bind=engine)
+
+s = Session()

```

Comment:
Using a global object to initialize and reference the session doesn't seem to be the best practice.

How about we do this?

```
from contextlib import contextmanager

# context manager explained here: https://docs.sqlalchemy.org/en/13/orm/session_basics.html
@contextmanager
def session_scope():
    """Provide a transactional scope around a series of operations."""
    session = Session()
    try:
        yield session
        session.commit()
    except:
        session.rollback()
        raise
    finally:
        session.close()
```

And then, for each function, we put the code within a `with session_scope() as session:` block like this:
```
def updateRelease(id, property, newValue):
    with session_scope() as session:
        rel = session.query(Release).get(id)
        setattr(rel, property, newValue)
```

This approach will make sure that, BEFORE the code within the `with` block is executed, the session will be created... and AFTER the code is executed, the session will trigger the COMMIT against the database to finalize the transaction and CLOSE the session afterwards (and, if anything goes wrong, it will rollback the transaction).